target_chains/ton/contracts/contracts/Pyth.fc

```diff
@@ -14,22 +40,25 @@ const int OP_EXECUTE_GOVERNANCE_ACTION = 2;
     return (price, conf, expo, publish_time);
 }
 
-(int, int, int, int) price_unsafe(int price_feed_id) method_id {
+(int, int, int, int) get_price_unsafe(int price_feed_id) method_id {
+    load_data();
     (slice result, int success) = latest_price_feeds.udict_get?(256, price_feed_id);
     throw_unless(ERROR_PRICE_FEED_NOT_FOUND, success);
     slice price_feed = result~load_ref().begin_parse();
     slice price = price_feed~load_ref().begin_parse();
     return parse_price(price);
 }
 
-(int, int, int, int) price_no_older_than(int time_period, int price_feed_id) method_id {
-    (int price, int conf, int expo, int publish_time) = price_unsafe(price_feed_id);
+(int, int, int, int) get_price_no_older_than(int time_period, int price_feed_id) method_id {
```

Comment:
rename the functions and add a `get_` prefix so as to follow the same standard in ethereum/starknet